### DISCUSSION TITLE
Looking to contribute my HTML & CSS knowledge to open source projects

### DISCUSSION THREAD

**Discussion by AlexandraGoldberg** [community/community#60973] [Programming Help]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I am looking to contrubute my HTML/CSS knowledge to open source project, I did not succeed on my own, please assist.

Alex

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by CypherpunkSamurai** [comment]
Use the GitHub search to narrow down issues, then change the language to your favourite. Usually ones that have "good-first-issue" tag are great friendly issues.

Alternatively you can also use this:
https://goodfirstissue.dev/

**Reply by AlexandraGoldberg** [reply]
Thanks a lot!!!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @AlexandraGoldberg ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects.

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
